### ISSUE TITLE
Add additional output layers with less specific codes.

### ISSUE THREAD

**Issue by tschimbr** [eonum/medcodelearn#24]
In the Graph model, once implemented, add additional softmax output classification layers with targets cropped to a certain length. 
The idea is that
1. We'll have an additional classifier for codes in higher hierarchies.
2. We can provide more information to the net (hierarchical topology of classes) and hence hopefully improve accuracy on initial output layer with the most specific classes.

**Comment by tschimbr** [comment]
Wait until Keras 1.0 is out with the new Graph API.

**Comment by tschimbr** [comment]
See https://github.com/eonum/medcodelearn/tree/hierarchical-targets

**Comment by tschimbr** [comment]
Improvement in overall accuracy was 0.23% and hence not significant.
